docs/pages/tutorial/eas/android-production-build.mdx

```diff
@@ -0,0 +1,318 @@
+/>
+
+- Return to the Google Play Console and go to **App bundles**. Click on **Upload** to add the **.aab**.
+- Provide the release details for our app and click on **Next**.
+- On the following screen, click on **Save and publish**.
+
+<ImageSpotlight
+  alt="Uploading aab file to Google Play Console"
+  src="/static/images/tutorial/eas/play-store-08.png"
+  style={{ maxWidth: 800 }}
+/>
+
+</Step>
+
+<Step label="6">
+
+## Share the internal release version
+
+Under **Track Summary**, we see that the latest release shows a temporary app name. This is because our app is not reviewed yet.
+
+<ImageSpotlight
+  alt="App pending review on Google Play Console"
+  src="/static/images/tutorial/eas/play-store-09.png"
+  style={{ maxWidth: 800 }}
+/>
+
+Under **Releases**, we see that the app is available to internal testers. To share the app with a team of testers:
+
+- Open the Internal testing dashboard, then click on **View release details**.
+- Click on **copy link** under **How testers join your test**.
+
+<ImageSpotlight
+  alt="Sharing the internal release version"
+  src="/static/images/tutorial/eas/play-store-14.png"
+  style={{ maxWidth: 800 }}
+/>
+
+- On the device, open the test email and follow the steps to download the app.
+
+<ImageSpotlight
+  alt="Downloading app on the device as an internal tester"
+  src="/static/images/tutorial/eas/play-store-15.png"
+  style={{ maxWidth: 280 }}
+/>
+
+- The testing email holder needs to accept the invite, and once accepted, the app can be installed on the device.
+
+<ImageSpotlight
+  alt="On accepting the invite, the app can be installed on the device"
+  src="/static/images/tutorial/eas/play-store-16.png"
+  style={{ maxWidth: 280 }}
+/>
+
+<ImageSpotlight
+  alt="App installed on the device as an internal tester"
+  src="/static/images/tutorial/eas/play-store-17.png"
+  style={{ maxWidth: 300 }}
+/>
+
+> **info** **Tip**: To publish an app on the Play Store, in the Google Dashboard, finish the steps under **Set up your app**. These steps are required before releasing the app on the Play Store for the first time. You'll have to provide details like a link to a privacy policy, a target audience, data safety and so on.
+
+> **Complete app store listing**: To prepare the app for store listing, see [Create app store assets](/guides/store-assets/) on how to create screenshots and previews.
+
+<Collapsible summary="Promoting a testing release">
+
+To promote our internal test release version to **alpha**, in Google Play Store Console:
+
+- Go to internal testing and click **Promote release**.
+- Open the dropdown menu and click **Closes testing** > **Closed testing - Alpha**.
+
+</Collapsible>
+
+</Step>
+
+<Step label="7">
+
+## Add Google Service Account permissions key
+
+From now on, we can use [EAS Submit](/submit/introduction/) to automate releases and avoid the manual process. To do that, we need to add the service account key to our project's **eas.json**.
```

Comment:
You can only do this if you fill out _all_ the metadata (including app assets etc) and it also has to go through a review process. Maybe this should be in a separate lesson?

Reply:
We've a callout and a tip about this before this section starts (which I think will help users to pay attention to.):

![CleanShot 2024-05-07 at 01 03 05](https://github.com/expo/expo/assets/10234615/c2f91389-e6dc-4395-aa95-cc769598b7df)